scripts/download-issues.sh

```
@@ -0,0 +1,44 @@
+
+# Create output directory if it doesn't exist
+mkdir -p "$OUTPUT_DIR"
+
+echo "Downloading issues from $OWNER/$NAME..."
+
+# Use gh CLI to fetch issues (handles auth automatically)
+gh issue list --repo "$OWNER/$NAME" --state all --limit 100 --json number,title,body | \
+  jq -c '.[]' | \
+while read -r ISSUE; do
+  NR=$(echo "$ISSUE" | jq -r '.number')
+  TITLE=$(echo "$ISSUE" | jq -r '.title')
```

Shellcheck (0.11.0)

[warning] TITLE appears unused. Verify use (or export if used externally).

(SC2034)